RecoTracker/MkFitCMS/src/MkSeedPartitioners-2017.cc

```diff
@@ -0,0 +1,355 @@
+#include "RecoTracker/MkFitCore/interface/Track.h"
```

Comment:
any reason why this has to contain 2017 in the name? wouldn't `-phase1` be more appropriate?

Reply:
We have 2017 nomenclature in a couple of places in MkFitXyz already, outside of this PR and also in the trackreco/mkFit-external where we have code to generate initial JSON files. Further, there is "2017:" used as a prefix for standard function names (seed/duplicate cleaning, seed partitioning, and quality filters).

As changing all 2017/2029 to phase1/2 will increase the scope of this PR I propose to open an issue and address all occurrences of run year consistently in function names, strings and file-names.

Reply:
I've been thinking -- it is probably easier for everyone to just do the full change here and now, for all file-names, identifiers, and strings.

Reply:
The rename has been done in https://github.com/cms-sw/cmssw/pull/39866/commits/75f87785d20e329bb5902c26c7c0461afd3a1f55